cmd/troubleshoot/cli/interactive_results.go

```diff
@@ -198,3 +316,19 @@ func save(analyzeResults []*analyzerunner.AnalyzeResult) (string, error) {
 
 	return filename, nil
 }
+
+func appName(supportBundleName string) string {
```

Comment:
This function looks to be copied from elsewhere? Should we move it to a shared package and use the same implementation?

Reply:
It is yes. Yep I can move it.